src/legacy/ui/public/visualize/components/visualization_chart.tsx

```diff
@@ -59,11 +55,6 @@ class VisualizationChart extends React.Component<VisualizationChartProps> {
     const render$ = this.renderSubject.asObservable().pipe(share());
 
     const success$ = render$.pipe(
-      tap(() => {
-        if (this.chartDiv.current) {
-          dispatchRenderStart(this.chartDiv.current);
-        }
-      }),
       filter(
```

Comment:
Charts don't fire `renderStart` anymore.
Is that intentional?
If so, handling of this event could be removed from `src/plugins/kibana_utils/public/render_complete/render_complete_helper.ts`

Reply:
yes, we don't use that helper in embeddalbe, we rather subscribe to expressionloader observables `lodaing$` and `render$`

Reply:
Cool, so that code can be deleted?

Reply:
render complete helper is still used by discover, so can't remove it yet.

Reply:
I meant not the whole file, just the relevant methods.

Reply:
Not critical anyway